packages/@ember/-internals/glimmer/lib/renderer.ts

```diff
@@ -239,7 +235,7 @@ export function renderSettled() {
     // a chance to resolve (because its resolved in backburner's "end" event)
     if (!_getCurrentRunLoop()) {
       // ensure a runloop has been kicked off
-      _backburner.schedule('actions', null, K);
```

Comment:
*Might* still make sense to have an extracted `const NO_OP = () => {}` here so we don't allocate a new one for every one of these, even if the name `K` is accursed forever.

Reply:
Do you think I should do this before merge? It seems unlikely that anyone else will go back and change it in the future if we don't do it now.

Reply:
Gut check, yes, for exactly that reason.